data/workflows.json

```diff
@@ -12,127 +12,359 @@
                 "label": "Evaluation Workflow dinglehopper_eval"
             },
             "gt_workspace": {
-                "@id": "https://github.com/OCR-D/quiver-data/blob/main/19_ant_simple_minimal_ocr.ocrd.zip",
-                "label": "GT workspace 19th century Antiqua simple layout"
+                "@id": "https://github.com/OCR-D/quiver-data/blob/main/16_frak_simple_selected_pages_ocr.ocrd.zip",
+                "label": "GT workspace 16th century Black letter simple layout"
             },
             "ocr_workspace": {
-                "@id": "https://github.com/OCR-D/quiver-back-end/blob/main/workflows/results/19_ant_simple_minimal_ocr_ocr.zip",
-                "label": "OCR workspace for 19_ant_simple_minimal_ocr"
+                "@id": "https://github.com/OCR-D/quiver-back-end/blob/main/workflows/results/16_frak_simple_selected_pages_ocr_ocr.zip",
+                "label": "OCR workspace for 16_frak_simple_selected_pages_ocr"
             },
             "eval_workspace": {
-                "@id": "https://github.com/OCR-D/quiver-back-end/blob/main/workflows/results/19_ant_simple_minimal_ocr_evaluation.zip",
-                "label": "Evaluation workspace for 19_ant_simple_minimal_ocr"
+                "@id": "https://github.com/OCR-D/quiver-back-end/blob/main/workflows/results/16_frak_simple_selected_pages_ocr_evaluation.zip",
+                "label": "Evaluation workspace for 16_frak_simple_selected_pages_ocr"
             },
             "workflow_steps": [
-                "ocrd-tesserocr-recognize"
+                {
+                    "ocrd-cis-ocropy-binarize": {
```

Comment:
It would be nice to have a bit more explicit object for a workflow step: 

```json
{
  "id": "ocrd-cis-ocropy-binarize",
  "params": {
      "method": "ocropy",
      "threshold": 0.5,
      "grayscale": false,
      "maxskew": 0.0,
      "noise_maxsize": 0,
      "dpi": 0,
      "level-of-operation": "page"
    }
}